src/dialog/overflow-menu/overflow-menu-pane.component.ts

```diff
@@ -119,6 +119,7 @@ export class OverflowMenuPane extends Dialog implements AfterViewInit {
 	}
 
 	protected listItems() {
-		return Array.from<any>(this.elementRef.nativeElement.querySelectorAll(".bx--overflow-menu-options__btn:not([disabled])"));
+		const selector = ".bx--overflow-menu-options__btn:not(.bx--overflow-menu-options__option--disable)";
+		return Array.from<any>(this.elementRef.nativeElement.querySelectorAll(selector));
```

Comment:
This would be `Array.from<Element>` since we know what type `querySelectorAll` returns ... if anything we may not even need to cast to `<any>` in the `.from`